build/OutputDirectories.props

```diff
@@ -25,5 +25,15 @@
       <TestPackagesDir>$(TestOutputDir)/packages/</TestPackagesDir>
       <TestArtifactsDir>$(TestOutputDir)/artifacts/</TestArtifactsDir>
       <TestResultXmlDir>$(TestOutputDir)/results/</TestResultXmlDir>
+      <ArchiveOutputDirectory>$(PackagesDirectory)</ArchiveOutputDirectory>
```

Comment:
Nevermind, I'm getting a test failure. I think I'll do the clean up now (and close this PR).